angular/src/app/components/main-welcome/main-welcome.component.styl

```diff
@@ -90,6 +90,12 @@
         padding-bottom 50px
         justify-content center
         align-items center
+      #welcome-banner
+        background-color: #fff3cd;
```

Comment:
Along with the stronger verbiage, could we make the BG color redish (maybe like `#ffd1cd`) so as to draw more attention to it? Yellow implies informational, this is more a warning of a mandatory step to be aware of.

Reply:
Since this is a message that won't clear if a user takes this mandatory step, I think it makes sense to keep it yellow and informational. We'll have to redeploy to remove the message down the line but there's nothing dynamic about it to discern if a user has taken the step or not.